developer/src/kmc-ldml/test/compiler-e2e.tests.ts

```diff
@@ -8,40 +9,59 @@ import { kmxToXml } from '../src/util/serialize.js';
 import { writeFileSync } from 'node:fs';
 import { LdmlCompilerMessages } from '../src/main.js';
 import { util } from '@keymanapp/common-types';
+import { KMXPlusVersion } from '@keymanapp/ldml-keyboard-constants';
+
+const debug=false;
 
 /** Overall compiler tests */
 describe('compiler-tests', function() {
   this.slow(500); // 0.5 sec -- json schema validation takes a while
 
-  before(function() {
+  beforeEach(function() {
     compilerTestCallbacks.clear();
   });
 
-  it('should-build-fixtures', async function() {
-    this.timeout(4000);
-    // Let's build basic.xml
-    // It should match basic.kmx (built from basic.txt)
-
-    const inputFilename = makePathToFixture('basic.xml');
-    const binaryFilename = makePathToFixture('basic.txt');
-
-    // Compare output
-    const expected = await hextobin(binaryFilename, undefined, {silent:true});
-
-    // now compare it to use with run()
-    // Let's build basic.xml
-    // It should match basic.kmx (built from basic.txt)
-    const k = new LdmlKeyboardCompiler();
-    await k.init(compilerTestCallbacks, { ...compilerTestOptions, saveDebug: true, shouldAddCompilerVersion: false });
-
-    const { artifacts } = await k.run(inputFilename, "basic-xml.kmx"); // need the exact name passed to build-fixtures
-    assert.isNotNull(artifacts);
-    const { kmx, kvk  } = artifacts;
-    assert.isNotNull(kmx);
-    assert.deepEqual<Uint8Array>(kmx?.data, expected);
+  afterEach(function() {
+    if (this.currentTest.state !== 'passed') {
+      compilerTestCallbacks.printMessages();
+    }
+  });
```

Comment:
This is cleanup that I am doing as I touch .tests.ts files in the kmc collection. Smooths out the individual tests by removing unnecessary logging there.